scripts/install_browser_driver.py

```diff
@@ -0,0 +1,122 @@
+import sys
+import zipfile
+
+import requests
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-u",
+    "--url",
+    dest="url",
+    required=True,
+    help="URL to browser or driver to download",
+)
+parser.add_argument(
+    "-c",
+    "--checksum",
+    dest="checksum",
+    required=True,
+    help="Checksum value to verify integrity of the product",
+)
+parser.add_argument(
+    "--unzip",
+    action="store_true",
+    default=True,
+    help="Unzip the resource",
+)
+parser.add_argument(
+    "-d",
+    "--destination",
+    dest="destination",
+    required=True,
+    help="Path to folder where the resource will be stored.Any missing folder will be created.",
+)
+
+
+class ChecksumVerifyError(Exception):
+    pass
+
+
+def download_resource(url, destination, checksum):
+    """
+    Download resource from the web and verify its integrity.
+
+    :param url: URL to the resource
+    :param destination: path to folder that will store the resource
+    :param checksum: pre-computed SHA256 checksum
+
+    :return: file path to downloaded resource as :class:`pathlib.Path`
+    """
+    dest = pathlib.Path(destination)
+    dest.mkdir(parents=True, exist_ok=True)
+    filepath = dest.joinpath(url.split('/')[-1])
+
+    print(f"Downloading {filepath.name} ...", end="")
+
+    try:
+        with requests.get(url, stream=True) as resp:
+            resp.raise_for_status()
+            with filepath.open('wb') as f:
+                for chunk in resp.iter_content(chunk_size=8192):
+                    f.write(chunk)
+
+        verify_checksum(filepath, checksum)
+    except Exception as err:
+        print(f"failed (reason: {err})")
+        sys.exit(1)
+    else:
+        print(f"done")
+
+    return filepath
+
+
+def verify_checksum(filepath, checksum):
+    """
+    Verify SHA256 checksum of a resource. Raises an :class:`ChecksumVerifyError` in case of mismatch.
+
+    :param filepath: :class:`pathlib.Path` path to resource
+    :param checksum: checksum to compare as hexadecimal :class:`str`
+    """
+    with filepath.open("rb") as f:
+        file_hash = hashlib.sha256()
+        while chunk := f.read(8192):
+            file_hash.update(chunk)
+
+    hexdigest = file_hash.hexdigest()
+    if hexdigest != checksum:
+        raise ChecksumVerifyError(f"checksum mismatch (digested: {hexdigest})")
+
+
+def unzip_resource(filepath):
+    """
+    Unzip a file in the same parent directory.
+
+    :param filepath: :class:`pathlib.Path` to file to unzip
+    """
+    print(f"Unziping {filepath.name}...", end="")
+    try:
+        with zipfile.ZipFile(filepath, 'r') as z:
+            z.extractall(path=filepath.parent)
+    except Exception as err:
+        print(f"failed (reason: {err})")
+    else:
+        print("done")
+
+
+if __name__ == "__main__":
+    args = parser.parse_args()
+
+    filepath = download_resource(args.url, args.destination, args.checksum)
+    if args.unzip:
+        unzip_resource(filepath)
```

Comment:
A permission issue. I guess Python `unzip()` doesn't work the same than linux `unzip` command.
Resolved anyway since I ditched this installation script.